packages/react-components/src/components/chart/converters/convertThresholds.ts

```diff
@@ -0,0 +1,66 @@
+import { AnnotationValue, COMPARISON_OPERATOR, ComparisonOperator, Threshold } from '@iot-app-kit/core';
+import { COMPARATOR_MAP } from '../../../common/constants';
+
+const comparisonOperatorToLowerYAxis = (comparisonOperator: ComparisonOperator, value: AnnotationValue) => {
+  switch (comparisonOperator) {
+    case COMPARISON_OPERATOR.GT:
+    case COMPARISON_OPERATOR.GTE:
+    case COMPARISON_OPERATOR.EQ:
+      return value;
+    case COMPARISON_OPERATOR.LT:
+    case COMPARISON_OPERATOR.LTE:
+      return 0;
+    default:
+      return undefined;
+  }
```

Comment:
I'm curious, is the default actually possible? If it's not possible, depending on your philosophy, this could be a great place to throw an error and declare an explicit return annotation as a union with `never`. What's going to happen is when a new operator added, this function will throw an error to tell the developer they need to extend this switch statement. Otherwise, its possible this function may be skipped if no immediate downstream impact is observed.